.changeset/thirty-years-drop.md

```diff
@@ -0,0 +1,11 @@
+---
+'@commercetools-uikit/select-input': patch
+'@commercetools-uikit/label': patch
+'@commercetools-uikit/link': patch
+'@commercetools-uikit/messages': patch
+'@commercetools-uikit/notifications': patch
+'@commercetools-uikit/pagination': patch
+'@commercetools-uikit/text': patch
+---
+
+Pagination component migrated to Typescript
```

Comment:
I think it'll be nice to have separate changesets for the packages and give information on what was changed for each package.

Reply:
Updated in this commit: cd354fe3d2e0fbdca6a6512866851f3ec6469a72

Thanks for the advice 👍